### PR TITLE
Added selection of reasons when user wants to delete his account.

### PR DESCRIPTION
**Pull request by jerdzica**

### CODE REVIEW

File: addons/bestja_account_deletion/views/report_account_deletion_wizard.xml [hunk 1]
```diff
@@ -18,14 +18,15 @@
                  </button>
             </field>
         </record>
-
+
```
Comment (KrzysiekJ) [inline addons/bestja_account_deletion/views/report_account_deletion_wizard.xml]: We should have no spaces at the end of line.

File: addons/bestja_account_deletion/wizards.py [hunk 2]
```diff
@@ -23,7 +23,16 @@ def delete_account(self):
...
+    REASONS = [
+        ('no_time', 'Nie mam czasu.'),
+        ('location_change', 'Zmieniam miejsce zamieszkania.'),
+        ('personal_change', 'Zmienia się moja sytuacja osobista (np. kończę studia).'),
+        ('bad_offers', 'Oferty nie spełniły moich oczekiwań.'),
+        ('no_satisfaction', 'Wolontariat nie sprawia mi już satysfakcji.'),
+        ('else', 'Inny (wpisz powód)'),
+    ] 
+    reason = fields.Selection(REASONS, string="Dlaczego chcesz usunąć konto?", required=True)
+    reason_description = fields.Char()
```
Comment (KrzysiekJ) [inline addons/bestja_account_deletion/wizards.py]: This should be set in the `reason_for_deleting_account` field when `reason` is “else”.

File: addons/bestja_account_deletion/wizards.py [hunk 3]
```diff
@@ -23,7 +23,16 @@ def delete_account(self):
 class ReportWizard(models.TransientModel):
     _name = 'bestja.report_account_deletion_wizard'
 
-    reason = fields.Text(string="Dlaczego chcesz usunąć konto?", required=True)
+    REASONS = [
```
Comment (ludwiktrammer) [inline addons/bestja_account_deletion/wizards.py]: I remember Ania saying user should be able to choose multiple reason. I think your solution is very reasonable, but can you confirm not allowing multiple choices is acceptable?

**Comment by ludwiktrammer** [comment]
You should change the `report_delete_account` accordingly, to save the value of both fields from wizard in the user model.

**Comment by KrzysiekJ** [comment]
Merged in ba83739.

**Pull request closed without merging**